src/ansys/aedt/core/filtersolutions_core/lumped_topology.py

```diff
@@ -34,26 +34,26 @@
 class LumpedTopology:
     """Defines topology parameters of lumped filters.
 
-    This class lets you construct all parameters for the ``LumpedDesign`` class.
+    This class allows you to define and modify the topology parameters of lumped filters.
     """
 
     def __init__(self):
         self._dll = ansys.aedt.core.filtersolutions_core._dll_interface()._dll
         self._dll_interface = ansys.aedt.core.filtersolutions_core._dll_interface()
         self._define_topology_dll_functions()
-        # self._set_lump_implementation()
+        self._set_lump_implementation()
 
     def _define_topology_dll_functions(self):
         """Define C++ API DLL functions."""
```

Comment:
@ramin4667 We need to add the logic here for this to work with both 25.1 and 25.2.

Reply:
Actually, I see that the solver tests passed. So the tests that would be affected must be skipped. We can address this later.